options.cpp

```diff
@@ -248,6 +248,8 @@ options::options()
        ("drill-speed", po::value<int>(), "spindle rpm when drilling")
        ("drill-front", po::value<bool>()->implicit_value(true), "[DEPRECATED, use drill-side instead] drill through the front side of board")
        ("drill-side", po::value<string>()->default_value("auto"), "drill side; valid choices are front, back or auto (default)")
+       ("drills-available", po::value<string>(), "List of available drills separated width ','. Tolerance can be specified with added '':<-tol>:<+tol>''")
```

Comment:
The type here should not be string, it should be `vector<available_drillbit>`.  You will need to override `operator>>` and `operator<<` for `vector<available_drillbit>`.  There are examples in units.hpp.  Look for the overrides for `Software`, for example.

Reply:
OK, I look into that.